framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -24,13 +24,19 @@
  * This class behaves similarly to javac. CheckerMain does the following:
  *
  * <ul>
- *   <li>add the {@code javac.jar} to the runtime classpath of the process that runs the Checker
+ *   <li>Add the {@code javac.jar} to the runtime classpath of the process that runs the Checker
  *       Framework.
- *   <li>add {@code jdk8.jar} to the compile time bootclasspath of the javac argument list passed to
- *       javac
- *   <li>parse and implement any special options used by the Checker Framework, e.g., using
- *       "shortnames" for annotation processors
- *   <li>pass all remaining command-line arguments to the real javac
+ *   <li>Use the annotated JDK.
+ *       <ul>
+ *         <li>If using Java 8: add {@code jdk8.jar} to the compile-time bootclasspath of the javac
+ *             argument list passed to javac.
+ *         <li>If using Java 9+: patch JDK modules using {@code --patch-module
+ *             <module>=<annotatedModule>}. Annotated modules are present in the {@code
+ *             checker/dist/annotatedJDK/jdk*\/} directory.
```

Comment:
Shouldn't this part depend on the `JDK_PATH_OPT` flag? `checker/dist` is just the default value for it. (Or maybe it works differently and you need to explain how it is supposed to work...)